crates/router/src/core/relay/utils.rs

```diff
@@ -32,7 +32,10 @@ pub async fn construct_relay_refund_router_data<'a, F>(
     let webhook_url = Some(payments::helpers::create_webhook_url(
         &state.base_url.clone(),
         merchant_id,
-        connector_name,
+        #[cfg(feature = "v1")]
+        &connector_account.merchant_connector_id,
+        #[cfg(feature = "v2")]
+        &connector_account.id,
```

Comment:
```suggestion
        &connector_account.get_id(),
```

Reply:
The `id` field is already public, is it necessary to use &connector_account.get_id() instead of &connector_account.id?

Reply:
I suggested `get_id()` since it would replace the passing of either `merchant_connector_id` or the `id` field based on feature flag.

Reply:
Sure, will make the suggested changes